### PR TITLE
Fix for #596

### PR DESCRIPTION
**Pull request by jokoho48**

### CODE REVIEW

File: serverAddons/AAW/addons/AAW_Server/FOB/fn_defuseAction.sqf [hunk 1]
```diff
@@ -17,9 +17,13 @@ private _title = "Defuse";
 private _iconIdle = "\A3\Ui_f\data\IGUI\Cfg\HoldActions\holdAction_forceRespawn_ca.paa";
 private _iconProgress = "\A3\Ui_f\data\IGUI\Cfg\HoldActions\holdAction_forceRespawn_ca.paa";
 private _showCondition = {
+
```
Comment (netfusion) [inline serverAddons/AAW/addons/AAW_Server/FOB/fn_defuseAction.sqf]: Remove this line

**Comment by netfusion** [comment]
https://github.com/TaktiCool/CLib/blob/master/addons/CLib/Interaction/fn_canInteractWith.sqf

**Comment by jokoho48** [comment]
normally it should already work. that is a point that also confused me. https://github.com/TaktiCool/CLib/blob/master/addons/CLib/Interaction/fn_addHoldAction.sqf

**Comment by netfusion** [comment]
Maybe cause CLib's `isNotDead` condition doesn't consider unconsciousness:
https://github.com/TaktiCool/CLib/blob/master/addons/CLib/Interaction/fn_clientInitCanInteractWith.sqf#L30

https://github.com/TaktiCool/ArmaAtWar/blob/master/serverAddons/AAW/addons/AAW_Server/Common/fn_isAlive.sqf

**Comment by jokoho48** [comment]
and that is fine but someone removed canInteract Conditions from Revive https://github.com/TaktiCool/ArmaAtWar/blob/master/serverAddons/AAW/addons/AAW_Server/Revive/fn_clientInit.sqf

**Comment by netfusion** [comment]
I would suggest adding a new condition `isNotUnconscious`

**Comment by jokoho48** [comment]
https://github.com/TaktiCool/ArmaAtWar/commit/f86fe02579f4ad7d58b3332bb466d92ce9a4f474 should be fix by that

**Pull request closed without merging**